src/main/java/org/wildfly/security/http/HttpServerScopes.java

```diff
@@ -27,6 +27,22 @@
 public interface HttpServerScopes {
 
     /**
+     * Returns {@code true} if the specified scope exists.
+     *
+     * @param scope the scope required.
+     * @return {@code true} if the specified scope exists. Otherwise, {@code false}
+     */
+    boolean exists(Scope scope);
```

Comment:
I've considered that initially but, currently, client code (using Scope API) is expecting null and not an empty instance. So I decided to avoid more changes and keep current behavior.

I totally agree with you, should we proceed with this refactoring then ?